scripts/Source/sslConfigMenu.psc

```diff
@@ -616,14 +635,24 @@ event OnSliderAcceptST(float value)
 	; Animation Editor
 	if Options[0] == "Adjust"
 		; Stage, Slot
-		Animation.SetAdjustment(AdjustKey, Position, Options[1] as int, Options[2] as int, value)
-		Config.ExportProfile(Config.AnimProfile)
-		if Options[2] == "3" ; SOS
-			SetSliderOptionValueST(value, "{0}")
+		if Config.MirrorPress(Config.AdjustStage) && ShowMessage("$SSL_WarnApplyAllStages", true, "$Yes", "$No")
```

Comment:
Allowed set same value on the current edited Adjustment for all the animation stages if the CTRL key is keeped pressed when accept.

Very useful when all or almost all the stages use the same value for the current actor.